ui/src/app/Pages/candidates/candidates.component.ts

```diff
@@ -0,0 +1,25 @@
+import { Component, OnInit } from '@angular/core';
+import { FormBuilder, FormGroup, Validators } from '@angular/forms';
+
+@Component({
+  selector: 'app-candidates',
+  templateUrl: './candidates.component.html',
+  styleUrls: ['./candidates.component.sass']
+})
+export class CandidatesComponent implements OnInit {
+
+  constructor(private fb: FormBuilder) { }
+  candidateForm : FormGroup;
+  ngOnInit(): void {
+    this.candidateForm = this.fb.group({
```

Comment:
make form in outside function and call it in ngOnInit 
to use single principle responsibility

Reply:
okay